### DISCUSSION TITLE
V200 is the issue with the liquid temp and Fancontrol freezing now resolve?

### DISCUSSION THREAD

**Discussion by SnowMN253** [Rem0o/FanControl.Releases#2703] [Q&A]
Good afternoon,

Read the patch notes just wanted to make sure if the issue is now resolved .

Thanks guys!

**Comment by Rem0o** [comment]
What issue? If you mean https://github.com/Rem0o/FanControl.Releases/issues/2658, then no this wasn't addressed.

**Comment by SnowMN253** [comment]
No, I edited and closed that one thanks for the reminder,here I was refering to the issue where software froze when using liquid TCL, but now I used the hardware library all and I have no more issues,tha KS for getting back to me none the less.
